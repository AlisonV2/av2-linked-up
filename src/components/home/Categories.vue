<template>
  <div class="row">
    <app-title>Recherche par style</app-title>
    <div class="col-12 col-md-6 col-lg-3">
      <div class="card">
        <img src="blackwork.jpg" class="card-img-top" alt="..." />
        <div class="card-body">
          <h5 class="card-title">Black Work</h5>
        </div>
      </div>
    </div>
    <div class="col-12 col-md-6 col-lg-3">
      <div class="card">
        <img src="oldschool.jpg" class="card-img-top" alt="..." />
        <div class="card-body">
          <h5 class="card-title">Old School</h5>
        </div>
      </div>
    </div>
    <div class="col-12 col-md-6 col-lg-3">
      <div class="card">
        <img src="newschool.jpg" class="card-img-top" alt="..." />
        <div class="card-body">
          <h5 class="card-title">Surr√©alisme</h5>
        </div>
      </div>
    </div>
    <div class="col-12 col-md-6 col-lg-3">
      <div class="card">
        <img src="surrealism.jpg" class="card-img-top" alt="..." />
        <div class="card-body">
          <h5 class="card-title">New School</h5>
        </div>
      </div>
    </div>
  </div>
</template>

<script>
export default {};
</script>

<style lang="scss">
.card-title {
  text-align: center;
  font-size: 1rem;
}

.card {
  border-radius: 4px;
  box-shadow: 0 6px 10px rgba(0, 0, 0, 0.08), 0 0 6px rgba(0, 0, 0, 0.05);
  transition: 0.3s transform cubic-bezier(0.155, 1.105, 0.295, 1.12),
    0.3s box-shadow;
  cursor: pointer;
  img {
    height: 12rem;
  }
}

.card:hover {
  transform: scale(1.05);
  box-shadow: 0 10px 20px rgba(0, 0, 0, 0.12), 0 4px 8px rgba(0, 0, 0, 0.06);
}
</style>
