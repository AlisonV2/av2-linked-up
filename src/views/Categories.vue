<template>
  <div class="container">
    <HomeCategories />
  </div>
</template>

<script>
import HomeCategories from '@/components/home/Categories';

/**
 * @exports Categories
 * @type {Page}
 * @requires HomeCategories
 */
export default {
  name: 'Categories',
  components: {
    HomeCategories,
  },

};
</script>
