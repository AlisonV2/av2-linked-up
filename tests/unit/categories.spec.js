import { shallowMount } from '@vue/test-utils';
import Categories from '@/views/Categories.vue';
import HomeCategories from '@/components/home/Categories';

/**
 * @module CategoriesPageTest
 */
describe('Categories.vue', () => {
  it('Check if CategoriesGeoloc is rendered', () => {
    const component = shallowMount(Categories, {
      global: {
        components: {
          HomeCategories,
        },
      },
    });
    const cat = component.findComponent(HomeCategories);
    expect(cat.exists()).toBe(true);
  });
});
