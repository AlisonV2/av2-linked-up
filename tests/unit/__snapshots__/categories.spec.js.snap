// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Categories.vue Categories Snapshot 1`] = `
<div
  class="container"
>
  <home-categories-stub />
</div>
`;
